using_images/xpaas_images/a_mq.adoc

```diff
@@ -105,6 +115,61 @@ application's Git project root. On each commit, the file will be copied to the
 *_conf_* directory in the A-MQ root and its contents used to configure the
 broker.
 
+== Configuring the JBoss A-MQ Persistance Image
```

Comment:
s/Persistance/Persistence/

However, the nomenclature is "persistent" storage/volume/image. Looks like the AMQ image in question is also called that, specifically, too:

```
# oc get templates -n openshift | grep amq
amq6                                    Application template for ActiveMQ brokers.                                         12 (2 blank)      8
amq6-persistent                         Application template for ActiveMQ brokers using persistent storage.                13 (2 blank)      9
eap6-amq-persistent-sti                 Application template for EAP 6 A-MQ applications with persistent storage buil...   22 (7 blank)      11
eap6-amq-sti                            Application template for EAP 6 A-MQ applications built using STI.                  21 (7 blank)      10
```

Also [upstream](https://github.com/jboss-openshift/application-templates/tree/master/amq).